en/finding-sorting-and-cleaning-entries/groups.md

```diff
@@ -1,132 +1,268 @@
-* **union**. Displays entries contained in this group or its subgroups.
+The content of the group is updated dynamically whenever the database changes.
 
-## Viewing a group's entries, combining multiple groups
+#### Using a free-form search expression 
 
-Selecting a group shows the entries contained in that group by highlighting them and, depending on the settings \(accessible by clicking the **Settings** button\), moving them to the top of the list and/or selecting them. These options are identical to those available for the regular search.
+This is similar to the above, but rather than search for a single search term on a single field, a [search expression syntax](search.md#advanced) can be used.
+It supports logical operators (`AND`, `OR`, `NOT`) and allows searching multiple fields.
 
-When multiple groups are selected, either the union or the intersection of their content is shown, depending on the current settings. This allows to quickly combine multiple conditions, e.g. if you have a static group `Extremely Important` to which you assign all extremely important entries, you can view the extremely important entries in any other group by selecting both groups \(this requires to have **Intersection** selected in the settings\).
+**%%% screenshot here: grab of the group type %%%**
+
+For example, the search expression `keywords=regression and not keywords=linear` groups entries concerned with non-linear regression.
+
+The content of the group is updated dynamically whenever the database changes.
+
+#### Specified keywords
+
+**%%%Help needed: I do not understand this feature. Please, step in %%%**
+
+**%%%Help needed: The following might be relevant %%%**
+By clicking the **Automatically create groups for database** button, you can quickly create a set of groups appropriate for your database.
+This feature will gather all words found in a specific field of your choice, and create a group for each word.
+This is useful for instance if your database contains suitable keywords for all entries. By auto-generating groups based on the `keywords` field, you should have a basic set of groups at no cost.
+
+You can also specify characters to ignore, for instance commas used between keywords.
+These will be treated as separators between words, and not part of them.
+This step is important for combined keywords such as `laplace distribution` to be recognized as a single semantic unit. 
+(You cannot use this option to remove complete words. Instead, delete the unwanted groups manually after they were created automatically.)
+
+**%%% screenshot here: grab of the group type %%%**
+
+The content of the group is updated dynamically whenever the database changes.
+
+#### Using the cited entries of a LaTeX document
+
+The group contains the entries cited in a LaTeX document, based on its '.aux' file.
+The .aux file has to be specified.
+
+**%%% screenshot here: grab of the group type %%%**
+
+**%%%Help needed: True? When updated exactly? %%%**
```

Comment:
True. It's updated as soon as the aux file changes on the disk, i.e. usually after you run your latex compilation.

Reply:
Well, I tested it and could not find out when it was updated (except after restarting JabRef). Unfortunately, a latex compilation is not enough...